File: Tests/AndroidSDKTestApp/AndroidSDKTestApp/Delegates/OPCallDelegateImplementation.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.Util;
using Com.Openpeer.Delegates;
using Com.Openpeer.Javaapi;
namespace AndroidSDKTestApp
{
	public class OPCallDelegateImplementation:OPCallDelegate
	{
		public override void OnCallStateChanged (OPCall call, CallStates state)
		{
			Log.Debug("output", "Call State = " + state.ToString());
		}
	}
}

File: openpeer-android-sdk/src/com/openpeer/sdk/delegates/OPCallDelegateImplementation.java
package com.openpeer.sdk.delegates;

import android.util.Log;

import com.openpeer.javaapi.CallStates;
import com.openpeer.javaapi.OPCall;
import com.openpeer.javaapi.OPCallDelegate;

public class OPCallDelegateImplementation extends OPCallDelegate {

	@Override
	public void onCallStateChanged(OPCall call, CallStates state) {
		Log.d("output", "onCallStateChanged " + state.toString() + " call "
				+ call);
	}

}
